lib/Cake/Cache/Cache.php

```diff
@@ -578,4 +578,41 @@ public static function remember($key, $callable, $config = 'default') {
 		return $results;
 	}
 
+/**
+ * Write data for key into a cache engine if it doesn't exist already.
+ *
+ * ### Usage:
+ *
+ * Writing to the active cache config:
+ *
+ * `Cache::add('cached_data', $data);`
+ *
+ * Writing to a specific cache config:
+ *
+ * `Cache::add('cached_data', $data, 'long_term');`
+ *
+ * @param string $key Identifier for the data
```

Comment:
Well we didn't go so far, to actually make a tiny dot at the end mandatory :trollface: but here you can see an example how we would like it:

http://book.cakephp.org/2.0/en/contributing/cakephp-coding-conventions.html#method-function-docblocks

And yes, not all pre-existing code does follow every timy bit of our coding standard. :smirk_cat: